doxygen/examples/H5.format.html

```diff
@@ -11523,6 +11543,119 @@ <h4><a name="DatatypeMessage">IV.A.2.d. The Datatype Message</a></h4>
                         </table>
                       </div>
 
+<br />
+<br />
+<a name="ClassComplex"></a>
+  <p>Class specific information for the Complex class (Class 11):</p>
```

Comment:
I suggest adding a version to this info, which will make supporting heterogenous complex types easier if they are added in the future.